app/assets/javascripts/application.js

```diff
@@ -11,8 +11,8 @@
 // about supported directives.
 //
 //= require rails-ujs
-//= require jquery3
 //= require turbolinks
-//= require popper
-//= require bootstrap-sprockets
+//= require jquery/dist/jquery
+//= require popper.js/dist/popper
```

Comment:
Just for reference, the npm installed Bootstrap actually still uses Tether, so I switched out the dependency.